Sources/VaporSQLite/Provider.swift

```diff
@@ -1,51 +1,54 @@
 import Vapor
 import FluentSQLite
 
-public typealias MySQLDriver = FluentSQLite.SQLiteDriver
-
 public final class Provider: Vapor.Provider {
-    public var provided: Providable
 
     
     public enum Error:Swift.Error {
         case noSQLiteConfig
         case pathMissing
     }
-    /**
-        MySQL database driver created by the provider.
-    */
-    public let driver: SQLiteDriver
-
-    /**
-        MySQL database created by the provider.
-    */
-    public let database: Database?
-
-    public init(
-        path: String
-    ) throws {
-        let driver = try SQLiteDriver(path: path)
-        let database = Database(driver)
-
-        self.driver = driver
-        self.database = database
-        
-        provided = Providable(database: database)
-    }
     
-    public convenience init(config: Config) throws {
+    private(set) public var driver: SQLiteDriver!
```

Comment:
I'm wondering if we actually need these properties, or if they'll be assumed to be set on the Droplet itself. Otherwise, I'm leaning towards having them be `?` instead of `!` since there's lots of ways that this could initialize w/o boot being called and these not being set.